EIPS/eip-1.md

```diff
@@ -82,7 +82,7 @@ The following is the standardization process for all EIPs in all tracks:
 
 If this period results in necessary normative changes it will revert the EIP to `Review`.
 
-**Final** - This EIP represents the final standard. A Final EIP exists in a state of finality and should only be updated to correct errata and add non-normative clarifications.
+**Final** - This EIP represents the final standard. A Final EIP exists in a state of finality and should only be updated to correct errata, add non-normative clarifications, or make backward compatible changes.
```

Comment:
I would prefer that backward compatible changes be made in new EIPs. Authors shouldn't be adding new features (that are backwards compatible) to their Final proposals without going through the review process.

Reply:
Fixed.